src/js/dashboard/dashboard.html

```diff
@@ -1,52 +1,48 @@
-                    <li class="active">
-                        <a ui-sref="logged.franklinRepos" class="dashboard__item">
-                            <svg class="dashboard__icon" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:sketch="http://www.bohemiancoding.com/sketch/ns" viewBox="0 0 100 100" version="1.1" x="0px" y="0px">
-                                <g stroke="none" stroke-width="1" sketch:type="MSPage">
-                                    <g sketch:type="MSArtboardGroup">
-                                        <path d="M45.5211765,62.4186765 C41.9917647,62.4186765 38.9864706,65.2951471 38.9864706,68.6939706 L38.9864706,86.4716176 C38.9864706,90.0010294 36.11,92.7451471 32.45,92.7451471 L15.0641176,92.7451471 C11.5347059,92.7451471 8.52941176,89.8704412 8.52941176,86.3410294 L8.52941176,48.3010294 C8.52941176,44.7716176 10.6205882,39.9345588 13.1035294,37.3210294 L44.0847059,6.86220588 C46.5676471,4.37926471 50.75,4.37926471 53.2347059,6.86220588 L84.2141176,37.3210294 C86.6988235,39.8039706 88.79,44.7716176 88.79,48.3010294 L88.79,86.3410294 C88.79,89.8704412 85.9135294,92.7451471 82.2535294,92.7451471 L66.4364706,92.7451471 C62.9070588,92.7451471 60.0323529,89.8704412 60.0323529,86.4716176 L60.0323529,68.6939706 C60.0323529,65.1645588 57.1558824,62.4186765 53.6264706,62.4186765 L45.5211765,62.4186765" sketch:type="MSShapeGroup" />
-                                    </g>
-                                </g>
-                            </svg>
-                            Dashboard</a>
-                    </li>
-                    <li>
-                        <a class="dashboard__item" href="" ng-click="dc.logout()">
-                            <svg class="dashboard__icon" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" x="0px" y="0px" viewBox="0 0 90 90" enable-background="new 0 0 90 90" xml:space="preserve">
-                                <g>
-                                    <path d="M83.7,43c0-0.3-0.1-0.6-0.2-0.9c-0.1-0.1-0.2-0.3-0.3-0.4c-0.1-0.1-0.1-0.2-0.2-0.3c0,0,0,0,0,0L70.3,28.6   c-0.9-0.9-2.3-0.9-3.2,0c-0.9,0.9-0.9,2.6,0,3.5l8.9,9.2H31.7c-1.2,0-2.3,1-2.3,2.3c0,1.2,1,2.3,2.3,2.3H76l-8.9,8.7   c-0.9,0.9-0.9,2.2,0,3.1c0.9,0.9,2.3,0.8,3.2-0.1l12.8-12.8c0,0,0,0,0,0c0.1-0.1,0.2-0.2,0.2-0.4c0.1-0.1,0.2-0.2,0.2-0.4   C83.7,43.6,83.7,43.3,83.7,43L83.7,43L83.7,43z" />
-                                    <path d="M53.9,58.8v17.6H13.2V13.1h40.7v16.5h4.5V13.1c0-2.5-1.9-4.5-4.2-4.5H12.9c-2.3,0-4.2,2-4.2,4.5v63.3   c0,2.5,1.9,4.5,4.2,4.5h41.3c2.3,0,4.2-2,4.2-4.5V58.8H53.9z" />
-                                </g>
+            <div class="columns small-3" style="border-left: 1px solid #3bd5df;">
+                <div class="dashboard__user-info">
+                    <div class="dashboard__user-image" style="background: url(https://avatars2.githubusercontent.com/u/8729714?v=3&s=460) no-repeat; background-size: contain;">
```

Comment:
It's an svg, only using it there for now, may be store in a file afterwards, we still don't have icons. Just got it from a open source project.